cfgov/templates/wagtailadmin/account/password_reset/confirm.html

```diff
@@ -26,23 +26,34 @@
                 </ul>
             </div>
         {% endif %}
-
-        <form method="post">
+        <form method="post" novalidate>
             {% csrf_token %}
             <h1>{% trans "Set your new password" %}</h1>
             <ul class="fields">
                 {% for field in form %}
                     <li class="{% if field.field.required %}required{% endif %} {{ wrapper_classes }} {{ li_classes }} {% if field.errors %}error{% endif %}">
-                      <div class="field {{ field|fieldtype }} {{ field|widgettype }} {{ field_classes }}">
+                    <div class="field {{ field|fieldtype }} {{ field|widgettype }} {{ field_classes }}">
```

Comment:
Do you know why we have this custom loop over fields instead of copying the explicit logic [in the Wagtail template](https://github.com/wagtail/wagtail/blob/v1.13.1/wagtail/wagtailadmin/templates/wagtailadmin/account/password_reset/confirm.html#L27)?

Reply:
Because of our custom password requirements. It's possible for a password to not have a sufficient character count, not have certain types of characters. We have more possible errors than Wagtail does.